pkg/backend/httpstate/policypack.go

```diff
@@ -150,7 +162,9 @@ func (pack *cloudPolicyPack) Apply(ctx context.Context, op backend.ApplyOperatio
 	return pack.cl.ApplyPolicyPack(ctx, pack.ref.orgName, string(pack.ref.name), op.Version)
 }
 
-func installRequiredPolicy(finalDir string, zip []byte) error {
+const npmPackageDir = "package"
+
+func installRequiredPolicy(finalDir string, tarball []byte) error {
```

Comment:
It's the .tar.gz file from `npm pack`. We call it `Unzip` because we refer to the process of un-tarring and unzipping as `Unzip`'ing ~everywhere in Pulumi. We could call it `Ungzip` and be slightly more correct.